com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JavaDebugDelegateCommandHandler.java

```diff
@@ -97,4 +102,19 @@ public Object executeCommand(String commandId, List<Object> arguments, IProgress
 
         return result;
     }
+
+    private boolean isOnClasspath(List<Object> arguments) {
+        if (arguments.size() < 1) {
+            return true;
```

Comment:
false?

Reply:
i prefer to throw an exception.

Reply:
agree, and it would be easier to find out any future abuse of the command (if there is)